readme.md

```diff
@@ -37,14 +37,32 @@ so you can focus on writing business logic.
 
 ## Quick start
 
+### Installation
```

Comment:
Alternative people on Linux can simply do `snap install ignite --classic`

Reply:
Thanks for the great suggestion @julienrbrt . The purpose of this PR was to resolve and issue raised about including the comments about creating the /usr/local/bin folder if we have an error. Do you want to add this? If so, please feel free to edit the readme.md, though please dont remove the comments I added as they work for both mac, linux and wsl.

Reply:
Right, but we should strive to have an easier what to install ignite.
Now we got snap for Linux / WSL and on macOS we'll have homebrew soon™️.

If we keep the path instructions for h to use that install ignite without package manager, then we should make it collapse by default as imo it clutters the readme.

Reply:
Thanks @julienrbrt, I am fine with whatever you like and dont want it too cluttered either. Would you mind editing it to your liking? So I can close [#2249](https://github.com/ignite/cli/issues/2249)

Cheers

Reply:
@julienrbrt I am thinking it might be best to update the readme.md with snap and brew installs when both are available. I was asigned that issue and it required a PR, so prefer to just get it done and we can revisit later if need be, or perhaps open a PR to make the changes you want?